app/controllers/ontologies_controller.rb

```diff
@@ -52,6 +52,7 @@ def create
 
   def show
     @content_object = :ontology
+    params[:id] = params[:ontology_id] if params[:ontology_id]
```

Comment:
Mhm, when using the `RefLocIdRouterConstraint`, `params[:id]` can be set to the version_id of the ontology versions as defined per https://github.com/ontohub/ontohub/blob/staging/lib/router_constraints.rb#L128. I'd assume that this can happen in this route: https://github.com/ontohub/ontohub/blob/staging/config/routes.rb#L36-L48. That sadly doesn't reveal an elegant solve, but might explain things ;)